.github/workflows/update.yml

```diff
@@ -1,8 +1,8 @@
-name: Update CellDeps Every 5 Minutes
+name: Update CellDeps Every 2 Minutes
 
 on:
   schedule:
-    - cron: '*/5 * * * *'  # Run every 5 minutes
+    - cron: '*/2 * * * *'  # Run every 2 minutes
```

Comment:
The shortest interval you can run scheduled workflows is once every 5 minutes.
See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule

Reply:
I‘m not sure. The cron examples show how to set 2 minutes as interval time.
See https://crontab.guru/examples.html